standalone/inc/pup/PUPTrigger.cpp

```diff
@@ -13,12 +13,13 @@ const char* PUP_TRIGGER_PLAY_ACTION_STRINGS[] = {
    "PUP_TRIGGER_PLAY_ACTION_STOP_FILE",
    "PUP_TRIGGER_PLAY_ACTION_SET_BG",
    "PUP_TRIGGER_PLAY_ACTION_PLAY_SSF",
-   "PUP_TRIGGER_PLAY_ACTION_SKIP_SAME_PRTY"
+   "PUP_TRIGGER_PLAY_ACTION_SKIP_SAME_PRTY",
+   "PUP_TRIGGER_PLAY_ACTION_CUSTOM_FUNC"
```

Comment:
for `142,1,TransSet,D0,15,"{""mt"":301, ""SN"":7, ""FN"":19, ""TM"":1, ""TC"":16777215}",,,,,,,CustomFunc,0`